server/controllers/api/config.ts

```diff
@@ -348,6 +353,11 @@ function customConfig (): CustomConfig {
     theme: {
       default: CONFIG.THEME.DEFAULT
     },
+    federation: {
+      videos: {
+        federateUnlisted: CONFIG.FEDERATION.VIDEOS.FEDERATE_UNLISTED
```

Comment:
Do you want admins to update this setting on the web interface? If no, you can remove this